lib/linguist/languages.yml

```diff
@@ -4554,6 +4554,19 @@ Raw token data:
   tm_scope: none
   ace_mode: text
   language_id: 318
+ReScript:
+  type: programming
+  color: "#df4b37"
+  ace_mode: rust
+  codemirror_mode: rust
+  codemirror_mime_type: text/x-rustsrc
+  extensions:
+  - ".res"
+  - ".resi"
```

Comment:
This extension can't be added yet as there are only 37 such files which is far from the "hundreds or repositories" requirement.